presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorIntegrationSmokeTest.java

```diff
@@ -193,4 +197,81 @@ public void testShowCreateTable()
         actualResult = computeActual("SHOW CREATE TABLE \"test_show_create_table\"\"2\"");
         assertEquals(getOnlyElement(actualResult.getOnlyColumnAsSet()), createTableSql);
     }
+
+    @Test
+    public void testTablesSystemTable()
+    {
+        assertUpdate("" +
+                "CREATE TABLE system_tables_test0 (c00 timestamp, c01 varchar, c02 double, c03 bigint, c04 bigint)");
+        assertUpdate("" +
+                "CREATE TABLE system_tables_test1 (c10 timestamp, c11 varchar, c12 double, c13 bigint, c14 bigint) " +
+                "WITH (temporal_column = 'c10')");
+        assertUpdate("" +
+                "CREATE TABLE system_tables_test2 (c20 timestamp, c21 varchar, c22 double, c23 bigint, c24 bigint) " +
+                "WITH (temporal_column = 'c20', ordering = ARRAY['c21', 'c22'])");
```

Comment:
Make the ordering `c22, c21` so that they are in the opposite order of the table declaration